### ISSUE TITLE
streaming url not working 

### ISSUE THREAD

**Issue by NormVg** [ytdl-org/youtube-dl#32146]
youtube-dl -g https://www.youtube.com/watch?v=uq3SUrhX2bk
ERROR: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2021.12.17**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--verbose']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Python version 3.10.8 (CPython) - Linux-5.15.0-1032-gcp-x86_64-with-glibc2.35
[debug] exe versions: none
[debug] Proxy map: {}
Usage: youtube-dl [OPTIONS] URL [URL...]

youtube-dl: error: You must provide at least one URL.
Type youtube-dl --help to see a list of all options.
```

## Description

WRITE DESCRIPTION HERE

**Comment by NormVg** [comment]
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--verbose']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Python version 3.10.8 (CPython) - Linux-5.15.0-1032-gcp-x86_64-with-glibc2.35
[debug] exe versions: none
[debug] Proxy map: {}
Usage: youtube-dl [OPTIONS] URL [URL...]

youtube-dl: error: You must provide at least one URL.
Type youtube-dl --help to see a list of all options.

**Comment by dirkf** [comment]
What use was that log?

Anyhow, your problem is solved by reading #30839 (**again**?) and/or by doing this properly:
>[x] I've searched the bugtracker for similar issues including closed ones
